k8s/k8s_test.go

```diff
@@ -14,7 +19,26 @@ func Test_ParseVulneDB(t *testing.T) {
 	var bi CVE
 	err = json.Unmarshal(b, &bi)
 	assert.NoError(t, err)
-	kvd, err := ParseVulnDBData(bi, map[string]string{})
+
+	mux := http.NewServeMux()
+	err = filepath.WalkDir("./testdata/mitreCVEs", func(path string, d fs.DirEntry, err error) error {
+		if !d.IsDir() {
+			f, err := os.ReadFile(path)
+			assert.NoError(t, err)
+			mux.HandleFunc(fmt.Sprintf("/%s", filepath.Base(path)), func(w http.ResponseWriter, r *http.Request) {
+				_, err = w.Write(f)
+				assert.NoError(t, err)
+			})
+		}
+		return nil
+	})
```

Comment:
nit: `err error` and `r *http.Request` are not in use 
```suggestion
	err = filepath.WalkDir("./testdata/mitreCVEs", func(path string, d fs.DirEntry, err error) error {
		if !d.IsDir() {
			var f []byte
			f, err = os.ReadFile(path)
			assert.NoError(t, err)
			mux.HandleFunc(fmt.Sprintf("/%s", filepath.Base(path)), func(w http.ResponseWriter, _ *http.Request) {
				_, err = w.Write(f)
				assert.NoError(t, err)
			})
		}
		return nil
	})
```